index.js

```diff
@@ -63,17 +63,12 @@ app.post('/register', function(request, response) {
 app.post('/messenger_auth', function(request, response) {
   //Extract the request received from Facebook
   var message = request.body.entry[0].messaging[0];
-
   console.log(message);
-
-  //Let's find out the first name of the user so that we can notify him/her using that later
-  fb = new FB.Facebook({});
-  FB.api('/' + message.sender.id, {access_token: env.FACEBOOK_PAGE_ACCESS_TOKEN}, function(resp){
-    var identity = resp.first_name;
-    var endpoint = 'FBM@' + identity;
-    //Let's create a new facebook-messenger Binding for our user
-    createBinding(identity, endpoint, 'facebook-messenger', message.sender.id, response);
-  });
+  // Set user identity using their fb id
+  var identity = message.recipient.id;
```

Comment:
Thanks for clarifying! I really do like the 3rd option, but I think it's a little out of scope right now - maybe we can add it as a 'nice to have' for the launch? I do like giving users a little more real-life context around building.

For now, my preference is to use message.sender.id as it's going to read clearly and 'just work' for people copy/pasta-ing code.